bucky/src/AddTwoNumbers.java

```diff
@@ -0,0 +1,16 @@
+import java.util.Scanner;
+
+public class AddTwoNumbers {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner (System.in);
+        System.out.println("first number");
+        int a = scanner.nextInt();
```

Comment:
change variable name from `a` to `firstNumber`

Reply:
done